### ISSUE TITLE
[Tiberian Sun] - Resource layer growth is blocked by elevation changes.

### ISSUE THREAD

**Issue by ABrandau** [OpenRA/OpenRA#15068]

**Comment by GraionDilach** [comment]
Do note that this was how TS worked, though. If anything, this needs to be made optional.

**Comment by pchote** [comment]
I believe we do need this for veins.

**Comment by ABrandau** [comment]
That is correct, veins are usually blocked because the veinhole monster is placed on a tile below, I would also suggest you to duplicate that actor but without the elevation changes, so people can place it on normal ground.
